io.ente.auth.yml

```diff
@@ -0,0 +1,227 @@
+        config-opts:
+          - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+          - -DAVIF_CODEC_DAV1D=ON
+          - -DBUILD_SHARED_LIBS=ON
+        sources:
+          - type: archive
+            url: https://github.com/AOMediaCodec/libavif/archive/v1.0.4/libavif-1.0.4.tar.gz
+            sha256: dc56708c83a4b934a8af2b78f67f866ba2fb568605c7cf94312acf51ee57d146
+            x-checker-data:
+              type: json
+              url: https://api.github.com/repos/AOMediaCodec/libavif/releases/latest
+              tag-query: .tag_name
+              timestamp-query: .published_at
+              version-query: $tag | sub("^[vV]"; "")
+              url-query: '"https://github.com/AOMediaCodec/libavif/archive/\($tag)/libavif-\($version).tar.gz"'
+      - name: libjxl
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+          - -DBUILD_TESTING=OFF
+          - -DJPEGXL_ENABLE_DEVTOOLS=OFF
+          - -DJPEGXL_ENABLE_TOOLS=OFF
+          - -DJPEGXL_ENABLE_DOXYGEN=OFF
+          - -DJPEGXL_ENABLE_MANPAGES=OFF
+          - -DJPEGXL_ENABLE_BENCHMARK=OFF
+          - -DJPEGXL_ENABLE_EXAMPLES=OFF
+          - -DJPEGXL_ENABLE_JNI=OFF
+          - -DJPEGXL_ENABLE_SJPEG=OFF
+          - -DJPEGXL_ENABLE_OPENEXR=OFF
+          - -DJPEGXL_ENABLE_SKCMS=OFF
+          - -DJPEGXL_FORCE_SYSTEM_BROTLI=ON
+          - -DJPEGXL_FORCE_SYSTEM_LCMS2=ON
+          - -DJPEGXL_FORCE_SYSTEM_HWY=ON
+        post-install:
+          # It doesn't respect disabling static libraries...
+          - rm $FLATPAK_DEST/lib/*.a
+        sources:
+          - type: archive
+            url: https://github.com/libjxl/libjxl/archive/v0.10.3/libjxl-0.10.3.tar.gz
+            sha256: e0191411cfcd927eebe5392d030fe4283fe27ba1685ab7265104936e0b4283a6
+            x-checker-data:
+              type: json
+              url: https://api.github.com/repos/libjxl/libjxl/releases/latest
+              tag-query: .tag_name
+              timestamp-query: .published_at
+              version-query: $tag | sub("^[vV]"; "")
+              url-query: '"https://github.com/libjxl/libjxl/archive/\($tag)/libjxl-\($version).tar.gz"'
+      - name: libwpe
+        buildsystem: meson
+        builddir: true
+        config-opts:
+          - --buildtype=plain
+        sources:
+          - type: archive
+            url: https://github.com/WebPlatformForEmbedded/libwpe/releases/download/1.16.0/libwpe-1.16.0.tar.xz
+            sha256: c7f3a3c6b3d006790d486dc7cceda2b6d2e329de07f33bc47dfc53f00f334b2a
+            x-checker-data:
+              type: json
+              url: https://api.github.com/repos/WebPlatformForEmbedded/libwpe/releases/latest
+              tag-query: .tag_name
+              timestamp-query: .published_at
+              version-query: $tag
+              url-query: '"https://github.com/WebPlatformForEmbedded/libwpe/releases/download/\($tag)/libwpe-\($version).tar.xz"'
+      - name: wpebackend-fdo
```

Comment:
same

Reply:
you only need libsoup 2.4, unifdef to build webkit2gtk https://github.com/flathub/net.sourceforge.liferea/blob/5ca27706e71b1f65e52b68597797ca00413f698d/net.sourceforge.liferea.json#L98-L144

Reply:
Are you sure about that? I think the mentioned recipe is for GNOME Platform and some deps might be missing.

Reply:
since most of your deps are there you should switch to `org.gnome.Platform`, libsecret, libkxl are already there. 

There's no point in building things twice.